tests/phpunit/payment-methods/class-wc-stripe-upe-payment-gateway-test.php

```
@@ -4190,4 +4193,135 @@ public function test_add_currency_conversion_notice_reads_from_order_meta_withou
+	 */
+	public function test_add_email_currency_conversion_notice_outputs_nothing_when_no_checkout_session(): void {
+		$order = WC_Helper_Order::create_order();
+		$order->set_payment_method( WC_Stripe_UPE_Payment_Gateway::ID );
+		$order->save();
+
+		ob_start();
+		$this->mock_gateway->add_email_currency_conversion_notice( $order );
+		$output = ob_get_clean();
+
+		$this->assertEmpty( $output );
+	}
```

PHPMD (2.15.0)

[error] The method test_add_email_currency_conversion_notice_outputs_nothing_when_no_checkout_session is not named in camelCase. (undefined)

(CamelCaseMethodName)